src/main/java/com/maxi/calendar/controller/EventController.java

```diff
@@ -0,0 +1,153 @@
+package com.maxi.calendar.controller;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.maxi.calendar.model.Event;
+import com.maxi.calendar.model.User;
+import com.maxi.calendar.service.IEventService;
+import com.maxi.calendar.service.IUserService;
+
+@RestController
+@RequestMapping("/events")
+public class EventController {
+	
+	@Autowired
```

Comment:
you should be using Constructor dependency injection. Check about the differences using variable injection

Reply:
@NoArgsConstructor lombock annotation

Reply:
the previous fix was incorrect, then I change to:
EventController (EventServiceImpl eventService, UserServiceImpl userService){
		this.eventService=eventService;
		this.userService=userService;
	}